about.html

```diff
@@ -0,0 +1,153 @@
+				<li><a href="poems-subject.html">Subject Matter</a></li>
+				<li><a href="#">Popularity</a></li>
+				
+				<li><a href="#">Location</a></li>
+			  </ul>
+			</li>
+			<li><a href="#">Magazine</a></li>
+			<li class="active"><a href="about.html">About</a></li>  
+			<!-- <li class="dropdown">
+			  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Poems <span class="caret"></span></a>
+			  <ul class="dropdown-menu">
+				<li><a href="#">Subject Matter</a></li>
+				<li><a href="#">Aurther</a></li>
+				<li><a href="#">Something else here</a></li>
+				<li role="separator" class="divider"></li>
+				<li><a href="#">Separated link</a></li>
+				<li role="separator" class="divider"></li>
+				<li><a href="#">One more separated link</a></li>
+			  </ul>
+			</li> -->
+		  </ul>	
+          <ul class="nav navbar-nav navbar-right">
+        		<li><a href="#"><span class="glyphicon glyphicon-user" aria-hidden="true"></span> Sign In</a></li>
+      	  </ul>  
+        </div><!--/.navbar-collapse -->
+      </div>
+    </nav>
+
+    <!-- Main jumbotron for a primary marketing message or call to action -->
+    <div class="jumbotron">
+      <div class="container">
+        <h1>ABOUT US</h1>— MegaPoem Creed<p></p>
+        <p>Sed dignissim lacinia nunc. Curabitur tortor. Pellentesque nibh. Aenean quam. In scelerisque sem at dolor. Maecenas mattis. Sed convallis tristique sem.</p>
+        
+      </div>
+    </div>
+
+    <div class="container container-home">
+      <!-- Example row of columns -->
+      <div class="row">
+        <div class="col-md-4">
+          <ul class="list-unstyled">
+			  
+			  <li><h3>Mission</h3><p>Sed dignissim lacinia nunc. Curabitur tortor. Pellentesque nibh. Aenean quam. In scelerisque sem at dolor. Maecenas mattis. Sed convallis tristique sem</p></li>
+			  <li><h3>The Team</h3>Integer euismod lacus luctus magna. Quisque cursus, metus vitae pharetra auctor, sem massa mattis sem, at interdum magna augue eget diam</li>
+			  <li><h3>Connect Us</h3>
+				  <a href="#"><i class="fa fa-github fa-2x" aria-hidden="true"></i></a>
+				  <a href="#"><i class="fa fa-facebook fa-2x" aria-hidden="true"></i></a>
+				  <a href="#"><i class="fa fa-twitter fa-2x" aria-hidden="true"></i></a>
+				  <a href="#"><i class="fa fa-google-plus fa-2x" aria-hidden="true"></i></a>
+			  </li>
+			  <li></li>	
+			</ul>
+        </div>
+        <div class="col-md-8">
+          <h2 class="visible-md-block visible-lg-block">Contact Us<br><br></h2>
+			<form>
+			  <div class="form-group">
+				<label>Your Name</label>
+				<input type="email" class="form-control" placeholder="your name">
+			  </div>
+			  <div class="form-group">
+				<label>Message</label>
+				<textarea class="form-control" rows="5" placeholder="your message goes here..."></textarea>
+			  </div>
+			  <button type="submit" class="btn btn-default">Send</button>
+			</form>	
```

Comment:
@JBigs206 I like this idea. A 50px bar across the top with a message would suffice.